index.js

```diff
@@ -1,23 +1,56 @@
 /**
- * @typedef {import('mdast').Root} Root
- * @typedef {import('retext-smartypants').Options} Options
+ * @typedef {import("mdast").Root} Root
+ * @typedef {import("retext-smartypants").Options} Options
  */
 
 import { retext } from "retext";
 import { visit } from "unist-util-visit";
 import smartypants from "retext-smartypants";
 
 /**
- * remark plugin to implement SmartyPants.
+ * Remark plugin to implement SmartyPants.
```

Comment:
Can you turn this back? I don't want to capitalize it, they keep lowercasing it so I want to as well.
```suggestion
 * remark plugin to implement SmartyPants.
```
<img width="500" alt="Screenshot 2023-12-31 at 11 25 32" src="https://github.com/silvenon/remark-smartypants/assets/471278/782b7297-e9ac-4c93-8963-d7f1642d01a7">